internal/fusekernel/fuse_kernel.go

```diff
@@ -199,6 +199,12 @@ func (fl OpenFlags) IsReadWrite() bool {
 	return fl&OpenAccessModeMask == OpenReadWrite
 }
 
+// Return true if OpenAppend is set.
+func (fl OpenFlags) IsAppend() bool {
+	flags:=fl &^ OpenAccessModeMask
```

Comment:
I don’t think removing the access mode bits is necessary. Please remove this line, the function can be implemented as `return fl&OpenAppend == OpenAppend`

Also, I see that your code is not formatted. Please always use `gofmt` to format code before sending a PR. Thanks!